ehr_analysis.py

```diff
@@ -4,51 +4,59 @@
 
 """Data Parsing"""
 
-def parse_patient(patient: str) -> List[List[str]]: 
-    with open(patient, "r") as file: 
-        rows = file.readlines() 
-        lists = [] 
-    for row in rows: 
-        row = row.strip() 
-        row = row.split("\t") 
+
+from datetime import datetime
+from typing import List
+
+
+def parse_patient(patient: str) -> List[List[str]]:
```

Comment:
We want these parsing functions to be of this form:
```suggestion
def parse_patient(filename: str) -> List[Patient]:
```
i.e. we can construct the `Patient` objects from the file data and return them.

Reply:
This issue is solved.